release-schema.json

```diff
@@ -63,6 +63,16 @@
           "codelist": "methodOfPayment.csv",
           "openCodelist": true,
           "uniqueItems": true
+        },
+        "payee": {
```

Comment:
Maybe order `payee` last so that payment method and address are closer together.